.github/workflows/test.yml

```diff
@@ -146,33 +146,34 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: 1.20.x
-      - name: Install GaiaV11
+      - name: Install Old Gaiad
         run: |
           git checkout v11.0.0
```

Comment:
Does the hardcoded `v11.0.0` version imply we have to make a change in a new release?

Reply:
yes, here need to be changed every time when we release a new major release, I can add a comment here. presently this test only runs on PR trying to merge to main. This means the test is targeting at from last major release to present PR branch binary version. I also tried:
1. config v11.0.0 by latest release or pre-release, it is messy depending on  latest release is before or after pre-release.
2. by using latest tag, somewhow this does not work in our repo, the latest tag is not the latest.